src/debug/ee/wks/CMakeLists.txt

```diff
@@ -61,6 +61,8 @@ elseif(CLR_CMAKE_PLATFORM_ARCH_ARM)
   add_library_clr(cordbee_wks ${CORDBEE_SOURCES_WKS} ../${ARCH_SOURCES_DIR}/dbghelpers.S)
 elseif(CLR_CMAKE_PLATFORM_ARCH_ARM64)
   add_library_clr(cordbee_wks ${CORDBEE_SOURCES_WKS})
+elseif(CLR_CMAKE_PLATFORM_ARCH_I386)
+  add_library_clr(cordbee_wks ${CORDBEE_SOURCES_WKS} ../${ARCH_SOURCES_DIR}/dbghelpers.S)
```

Comment:
Would you mind changing the `if` as follows so that the same body doesn't get repeated three times?
```cmake
if (CLR_CMAKE_PLATFORM_ARCH_AMD64 OR CLR_CMAKE_PLATFORM_ARCH_ARM OR CLR_CMAKE_PLATFORM_ARCH_I386)
  add_library_clr(cordbee_wks ${CORDBEE_SOURCES_WKS} ../${ARCH_SOURCES_DIR}/dbghelpers.S)
elseif()
.
.
```